implement-cowsay/cow.py

```diff
@@ -0,0 +1,20 @@
+import argparse
+import cowsay
+
+# Get the list of animals from cowsay
+available_animals = cowsay.char_names
+
+parser = argparse.ArgumentParser(
+    prog="cow.py",
+    description="Make animals say things"
+)
+
+parser.add_argument("--animal", choices=available_animals, default="cow", help="The animal to be saying things.")
```

Comment:
Non-blocking: you can format python code with various tools such as ruff and black, for example:
```
python -m pip install ruff
ruff format
```